Compilation warnings

I compiled epsilon to update my calculator on Debian Stretch.
I can see a warning alert, but flashing the calculator is ok.

[...]
```
CXX     poincare/src/expression.o
CXX     poincare/src/expression_lexer.o
poincare/src/expression_lexer.cpp: In function 'void yy_fatal_error(const char*)':
poincare/src/expression_lexer.cpp:2120:1: warning: 'noreturn' function does return
 }
 ^
At global scope:
cc1plus: warning: unrecognized command line option '-Wno-deprecated-register'
CXX     poincare/src/expression_parser.o
CXX     poincare/src/factorial.o
```
[...]

--- a/liba/include/stdlib.h
+++ b/liba/include/stdlib.h
@@ -10,7 +10,7 @@ void free(void *ptr);
 void * malloc(size_t size);
 void * realloc(void *ptr, size_t size);
 
-void abort(void);
+void abort(void) __attribute__((noreturn));
 
 LIBA_END_DECLS
 

